setup.py

```diff
@@ -33,14 +34,15 @@ def unique_flatten_dict(d):
     'networkx': ['networkx>=2.5'],
     'gremlin': ['gremlinpython'],
     'bolt': ['neo4j', 'neotime'],
-    'nodexl': ['openpyxl', 'xlrd'],
+    'nodexl': ['openpyxl==3.1.0', 'xlrd'],
     'jupyter': ['ipython'],
 }
 
 base_extras_heavy = {
   'umap-learn': ['umap-learn', 'dirty-cat==0.2.0', 'scikit-learn>=1.0'],
 }
-base_extras_heavy['ai'] = base_extras_heavy['umap-learn'] + ['scipy', 'dgl', 'torch', 'sentence-transformers', 'annoy', 'joblib']
+# https://github.com/facebookresearch/faiss/issues/1589 for faiss-cpu 1.6.1, #'setuptools==67.4.0' removed
```

Comment:
looks like an old issue that is no longer relevant?

Reply:
yes